lib/stdlib/doc/src/rand.xml

```diff
@@ -478,6 +523,68 @@ end.</pre>
     </func>
 
     <func>
+      <name name="uniform_real_s" arity="1"/>
```

Comment:
@okeuday This section tries to explain the properties and the value range of `uniform_real_s/1`.  As you see it is essentially [0.0,1.0) but not exactly 0.0, and I find it tricky to explain.